src/org/jitsi/impl/neomedia/rtp/MediaStreamTrackDesc.java

```diff
@@ -121,6 +121,39 @@ public MediaStreamTrackReceiver getMediaStreamTrackReceiver()
     }
 
     /**
+     * Gets the optimal bitrate (bps). Together with the current bitrate, it
+     * allows to calculate the probing bitrate.
+     *
+     * @return the optimal bitrate (bps).
+     */
+    public long getBps(int idx)
```

Comment:
why not call it `getOptimalBps`?  also, can you add a comment to define what the definition of 'optimal bitrate' is?